reader.py

```diff
@@ -0,0 +1,38 @@
+import http.client
+import json
+import datetime
+from util import SortUtil
+
+#Reads the data from the resources
+class Reader:
+    global start 
+    start = datetime.datetime.now()
+
+    def __init__(self, link, request):
+      self.link = link
+      self.request = request
+
+    #Establish connection and get data
+    def establish_connection(self, link, request):
+        print('----------Start----------')
+        print("Start Time: " + str(start))
+        conn = http.client.HTTPSConnection(self.link)
+        headers = {'Content-type': 'application/json'}
+        conn.request("GET", self.request, headers=headers)
+        response = conn.getresponse()
+        js = json.loads(response.read().decode())
+        conn.close() 
+        return js
+
+    #Sort data & print 
+    def sort_data(self, data):
```

Comment:
Sort data function should not be in this class.

Reply:
sort_data function has been moved to util.py